### ISSUE TITLE
Deploy to preprod takes latest version

### ISSUE THREAD

**Issue by nilsml** [equinor/energyvision#1102]
When deploying to preprod, the latest version is always built.
If Test is built and a new commit is pushed before Preprod is built, the build will fail as radixconfig and GHCR is not in sync. Also it is semantically wrong to deploy a different version to preprod than test.

**Comment by fernandolucchesi** [comment]
It looks good to me! 
However when I tried deploying my branch to dev, the logs seemed to be correct but my changes were not working in "dev". 
<img width="895" alt="image" src="https://user-images.githubusercontent.com/23337266/170708040-9c09ce58-86e0-4cac-aedc-b35cd8c102b4.png">
It could be that
1. My changes actually went through, but my code (branch that I tested) is buggy
2. Branch was not properly deployed

I am going for vacation but nice if someone can create a branch with a silly change and try to deploy it to confirm it works as intended.

I need to do some more testing before
